modules/react-native-zoom-free-calling/components/MeetingScheduleModal/index.js

```diff
@@ -0,0 +1,818 @@
+    { label: "1", value: "1" },
+    { label: "2", value: "2" },
+    { label: "3", value: "3" },
+    { label: "4", value: "4" },
+    { label: "5", value: "5" },
+    { label: "6", value: "6" },
+    { label: "7", value: "7" },
+    { label: "8", value: "8" },
+    { label: "9", value: "9" },
+    { label: "10", value: "10" },
+    { label: "11", value: "11" },
+    { label: "12", value: "12" },
+    { label: "13", value: "13" },
+    { label: "14", value: "14" },
+    { label: "15", value: "15" },
+    { label: "16", value: "16" },
+    { label: "17", value: "17" },
+    { label: "18", value: "18" },
+    { label: "19", value: "19" },
+    { label: "20", value: "20" }
+  ])
+  const [weeklyList, setWeeklyList] = useState([{
+    label: "Sun",
+    value: "1",
+    isSelected: true
+  }, {
+    label: "Mon",
+    value: "2",
+    isSelected: false
+  }, {
+    label: "Tue",
+    value: "3",
+    isSelected: false
+  }, {
+    label: "Wed",
+    value: "4",
+    isSelected: false
+  }, {
+    label: "Thu",
+    value: "5",
+    isSelected: false
+  }, {
+    label: "Fri",
+    value: "6",
+    isSelected: false
+  }, {
+    label: "Sat",
+    value: "7",
+    isSelected: false
+  }])
+  const [weeksList, setWeeksList] = useState([{
+    label: "First",
+    value: "1",
+    isSelected: true
+  }, {
+    label: "Second",
+    value: "2",
+    isSelected: false
+  }, {
+    label: "Third",
+    value: "3",
+    isSelected: false
+  }, {
+    label: "Fourth",
+    value: "4",
+    isSelected: false
+  }, {
+    label: "Last",
+    value: "-1",
+    isSelected: false
+  }])
+  const [daysList, setDaysList] = useState([{
+    label: "Sunday",
+    value: "1",
+    isSelected: true
+  }, {
+    label: "Monday",
+    value: "2",
+    isSelected: false
+  }, {
+    label: "Tuesday",
+    value: "3",
+    isSelected: false
+  }, {
+    label: "Wednesday",
+    value: "4",
+    isSelected: false
+  }, {
+    label: "Thursday",
+    value: "5",
+    isSelected: false
+  }, {
+    label: "Friday",
+    value: "6",
+    isSelected: false
+  }, {
+    label: "Saturday",
+    value: "7",
+    isSelected: false
+  }])
+  useEffect(() => {
+    let tmpRepeatEveryList = []
+    if (meetingSchedule.recurrence.recurrence_type == "1")
+      for (let i = 0; i < 15; i++) {
+        tmpRepeatEveryList.push({ label: (i + 1).toString(), value: (i + 1).toString() })
+      }
+    else if (meetingSchedule.recurrence.recurrence_type == "2")
+      for (let i = 0; i < 12; i++) {
+        tmpRepeatEveryList.push({ label: (i + 1).toString(), value: (i + 1).toString() })
+      }
+    else if (meetingSchedule.recurrence.recurrence_type == "3")
+      for (let i = 0; i < 3; i++) {
+        tmpRepeatEveryList.push({ label: (i + 1).toString(), value: (i + 1).toString() })
+      }
+    setRepeatEveryList(tmpRepeatEveryList)
+  }, [meetingSchedule.recurrence.recurrence_type])
+
+  useEffect(() => {
+    let tmpOccursOnList = []
+    for (let i = 0; i < 31; i++) {
```

Comment:
Same here, avoid magic numbers.

Reply:
When we schedule a recurrent meeting on zoom, it provides us static options to choose a time for the recurrent meeting i.e
from 1-15 days or 1-7 weeks 0r 1-3 months. The following depicts the same
![Screenshot from 2022-04-13 15-09-40](https://user-images.githubusercontent.com/76822297/163157403-e2d1739a-2ff5-4f61-9f37-e69436ddde57.png)
![Screenshot from 2022-04-13 15-10-11](https://user-images.githubusercontent.com/76822297/163157448-adf086a1-4d06-4d3a-9ac8-42196ffb2da6.png)
![Screenshot from 2022-04-13 15-10-47](https://user-images.githubusercontent.com/76822297/163157479-13fb6aa7-f1d3-475a-8485-e0f4f845496f.png)

I'll move these static numbers to options file